src/main/java/Duke.java

```diff
@@ -1,10 +1,155 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] tasks = new Task[100];
```

Comment:
Looks good, but need to pay attention to the code standard. 
It's better to add a space between the Task and [ ].